### ISSUE TITLE
Native Bytes resize bug

### ISSUE THREAD

**Issue by RobAustin** [OpenHFT/Chronicle-Bytes#37]
the log message always reports the  "Resizing buffer was" and "needs" as the same value which must be wrong.

```
[main] WARN net.openhft.chronicle.bytes.NativeBytes - Resizing buffer was 32364 KB, needs 32364 KB, new-size 48548 KB
[main] WARN net.openhft.chronicle.bytes.NativeBytes - Resizing buffer was 48548 KB, needs 48548 KB, new-size 72824 KB
[main] WARN net.openhft.chronicle.bytes.NativeBytes - Resizing buffer was 72824 KB, needs 72824 KB, new-size 109236 KB
------- 695271
[main] WARN net.openhft.chronicle.bytes.NativeBytes - Resizing buffer was 109236 KB, needs 109236 KB, new-size 163856 KB
[main] WARN net.openhft.chronicle.bytes.NativeBytes - Resizing buffer was 163856 KB, needs 163856 KB, new-size 245784 KB
```

chronicle-bytes version 1.7.34

**Comment by peter-lawrey** [comment]
This is warning that the resized buffers are getting very large. Do you
expect the buffer to grow to 254 MB?
ᐧ

**Comment by RobAustin** [comment]
the problem is that the 'was' and 'need' figure is the same value, "buffer was 163856 KB, needs 163856 KB", see log above for other examples.

**Comment by peter-lawrey** [comment]
Most likely this is a rounding error when it divides by 1024. It could be
changed to needs + X bytes more.

However you shouldn't be seeing this message in the first place. It
indicates there is a very large amount of data buffered up.
ᐧ

**Comment by RobAustin** [comment]
I've improved this log message to show the X bytes more

**Comment by hft-team-city** [comment]
Released in [Chronicle-Bytes-2.20.101](https://github.com/OpenHFT/Chronicle-Bytes/releases/chronicle-bytes-2.20.101), [BOM-2.20.134](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.20.134)
